docs/docs/authenticate/README.md

```diff
@@ -22,7 +22,7 @@ through an OIDC provider. The configuration file can be found at
 `/etc/firezone/firezone.rb`. To pick up changes, run `firezone-ctl reconfigure`
 and `firezone-ctl restart` to update the application.
 
-```ruby
+```shell
```

Comment:
Config file is in `ruby`

Reply:
@jamilbk `ruby` won't trigger proper syntax highlighting in docusaurus. Any idea why?

Reply:
Probably needs a plugin or something to be set up. Let me add a separate issue for it.